tests/e2e/test_rosacli_cluster.go

```diff
@@ -297,4 +297,216 @@ var _ = Describe("Edit cluster",
 				Expect(err).To(BeNil())
 				Expect(stdout.String()).To(ContainSubstring(fmt.Sprintf("Creating cluster '%s' should succeed", clusterName)))
 			})
+		It("Check the basic validation for the classic rosa cluster creation by the rosa cli - [id:38770]", labels.High, labels.NonHCPCluster, func() {
```

Comment:
Is it a day1 negative case? Define a new label with day1-validation and label these kind of cases.

Reply:
Validation cases maybe in importance of `Medium`?

Reply:
@xueli181114 Yes. It is a day-1 validation testing, however it needs the command of creating cluster which generate in job day-1 provision. If add such "day1-validation", will it be selected without day-q provision in future?